cocotb/decorators.py

```diff
@@ -385,24 +410,21 @@ class test(coroutine):
         stage (int, optional)
             Order tests logically into stages, where multiple tests can share a stage.
     """
-    def __init__(self, timeout=None, expect_fail=False, expect_error=False,
+    def __init__(self, f, timeout=None, expect_fail=False, expect_error=False,
```

Comment:
Could you add `f` to the documentation block?

Reply:
Not easily - `f` is not a real argument any more, it's the thing that appears under the `@cocotb.test`

Reply:
So let's treat `f` equal to `self` in terms of documentation.